ADBench/plot_graphs.py

```diff
@@ -387,14 +422,16 @@ def generate_graph(figure_idx, graph_function_type):
 
         labels.append(VIOLATION_LABEL)
 
-    build_type = graph[0]
-    maybe_test_size = graph[2:]
-    (graph_name, graph_save_location) = graph_data(build_type, objective, maybe_test_size, function_type)
+    (graph_name, graph_save_location) = graph_data(figure_info)
 
     # Setup graph attributes
+    xlabel = "No. independent variables"
+    if "hand" == figure_info.objective or "hand" in figure_info.maybe_test_size:
+        xlabel = "No. correspondencies"
```

Comment:
correspondencies -> correspondences

Reply:
Hmmm... interesting, it looks like this misprint has been being here for a long time)
Fixed.